specs/eventing/control-plane/channel-lifecycle/services.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: conformance-sockeye
+spec:
+  template:
+    spec:
+      containers:
+        - image: docker.io/n3wscott/sockeye:v0.7.0@sha256:e603d8494eeacce966e57f8f508e4c4f6bebc71d095e3f5a0a1abaf42c5f0e48
```

Comment:
we'll have to standardize on a single tool. @salaboy

Reply:
@lionelvillard I think we should yes.

Reply:
Are you using any other tool for this kind of tests? or we should discuss this in the WG meet?